rasa/cli/scaffold.py

```diff
@@ -15,16 +15,27 @@ def add_subparser(
     scaffold_parser = subparsers.add_parser(
         "init", parents=parents, help="Create a new project from a initial_project"
     )
+    scaffold_parser.add_argument(
+        "--no_prompt",
+        action="store_true",
+        help="Automatic yes or default options to prompts and " "suppressed warnings",
+    )
     scaffold_parser.set_defaults(func=run)
 
 
 def print_train_or_instructions(args: argparse.Namespace, path: Text) -> None:
     import questionary
 
-    print_success("Your bot is ready to go!")
-    should_train = questionary.confirm(
-        "Do you want me to train an initial model for the bot? 💪🏽"
-    ).ask()
+    print_success("Finished creating project structure.")
+
+    if not args.no_prompt:
+        should_train = questionary.confirm(
+            "Do you want me to train an initial " "model for the bot? 💪🏽"
+        ).ask()
+    else:
+        print_success("Training an initial model...")
```

Comment:
I think we should move the print after `if should_train` and use `.skip_if` for the confirm without the surrounding `if no args.no_prompt`